test/expected/insert_single.out

```diff
@@ -303,7 +303,7 @@ SELECT "1dim" FROM "1dim";
 
 --test that we can insert pre-1970 dates
 CREATE TABLE "1dim_pre1970"(time timestamp PRIMARY KEY, temp float);
-SELECT create_hypertable('"1dim_pre1970"', 'time');
+SELECT create_hypertable('"1dim_pre1970"', 'time', chunk_time_interval=> INTERVAL '1 Month');
```

Comment:
I would also test when chunk_time_interval given in numerical usec when time is a timestamp, timestamptz, date.

Reply:
Done